src/commands/publish.ts

```diff
@@ -273,14 +278,16 @@ export default class Publish extends Command {
 
     logger.success('publishSuccessful', documentType);
     let transition = true;
-    if (flags.force !== true) {
-      const prompt: { continue: boolean } = await inquirer.prompt({
-        name: 'continue',
-        message: `Do you want to switch to remote ${documentType} instead of a locally linked one?:`,
-        type: 'confirm',
-        default: true,
-      });
-      transition = prompt.continue;
+    if (flags.switch === true) {
```

Comment:
opinion: Although I find it kind of weird that we have a `--switch` flag which only enables the UI to ask whether to switch. I would expect to ask interactively unless the flag is specified, in which case it will decide whether to switch based on its argument (true/false). As far as I can tell right now there is no way to ask for the switch without interactivity, only to reject it beforehand (by not including the flag).

Reply:
So `--switch` => do not ask, just switch. No `--switch` - ask Interactively?

Reply:
Well, if I understand it correctly, there are three options:
1. Ask interactively (no `--switch` or possibly `--switch ask`)
2. Switch (`--switch` or `--switch always`)
3. Don't switch (`--no-switch` or `--switch never`)

Reply:
Yea, it depends if we want to cover the possible decisions by flags or we just want flag that prevents CLI from switching (and asking) that was kind first goal. But you are correct that using two flags is probably more future proof:)

Reply:
In that case just merge it as is, it fulfills the goal of being able to skip the interactive question 👍